cmd/query/app/static_handler.go

```diff
@@ -210,30 +211,43 @@ func loadIndexHTML(open func(string) (http.File, error)) ([]byte, error) {
 	return indexBytes, nil
 }
 
-func loadUIConfig(uiConfig string) (map[string]interface{}, error) {
+func loadUIConfig(uiConfig string) (*loadedConfig, error) {
 	if uiConfig == "" {
 		return nil, nil
 	}
 	ext := filepath.Ext(uiConfig)
-	bytes, err := ioutil.ReadFile(filepath.Clean(uiConfig))
+	bytesConfig, err := ioutil.ReadFile(filepath.Clean(uiConfig))
 	if err != nil {
 		return nil, fmt.Errorf("cannot read UI config file %v: %w", uiConfig, err)
 	}
-
-	var c map[string]interface{}
-	var unmarshal func([]byte, interface{}) error
+	var r []byte
 
 	switch strings.ToLower(ext) {
 	case ".json":
-		unmarshal = json.Unmarshal
+		var c map[string]interface{}
+
+		if err := json.Unmarshal(bytesConfig, &c); err != nil {
+			return nil, fmt.Errorf("cannot parse UI config file %v: %w", uiConfig, err)
+		}
+		r, _ = json.Marshal(c)
+
+		return &loadedConfig{
+			regexp: configPattern,
+			config: append([]byte("JAEGER_CONFIG = "), append(r, byte(';'))...),
+		}, nil
+	case ".js":
+		r = bytes.TrimSpace(bytesConfig)
+		if !bytes.Contains(r, []byte("function UIConfig(){")) {
+			return nil, fmt.Errorf("wrong JS function format in UI config file format %v", uiConfig)
```

Comment:
more self-explanatory error:
```suggestion
			return nil, fmt.Errorf("UI config file must define function UIConfig(): %v", uiConfig)
```